drivers/net/ksz9477_reg.h

```diff
@@ -0,0 +1,149 @@
+/****************************************************************************
+ * drivers/net/ksz9477_reg.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __DRIVERS_NET_KSZ9477_REG_H
+#define __DRIVERS_NET_KSZ9477_REG_H
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <nuttx/i2c/i2c_master.h>
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+/* These define only a subset of full register map. New registers can added
+ * as needed
+ */
+
+#define KSZ9477_I2C_ADDRESS                0x5f
+#define KSZ9477_I2C_SPEED                  I2C_SPEED_FAST
+
+/* Global registers */
+
+#define KSZ9477_ID0                        0x0
+#define KSZ9477_ID1                        0x1
+#define KSZ9477_ID2                        0x2
+#define KSZ9477_ID3                        0x3
+
+/* Port registers */
+
+#define KSZ9477_PORT_REG(port,x)           ((((uint16_t)port) << 12) | (x))
```

Comment:
(port) ?

Reply:
Yes, definitely. I wil also change x -> reg for clarity